tests/js/spec/views/performance/transactionSummary/transactionEvents/eventsTable.spec.tsx

```diff
@@ -0,0 +1,252 @@
+import {mountWithTheme} from 'sentry-test/enzyme';
+import {initializeOrg} from 'sentry-test/initializeOrg';
+
+import {t} from 'app/locale';
+import ProjectsStore from 'app/stores/projectsStore';
+import EventView from 'app/utils/discover/eventView';
+import {
+  SPAN_OP_BREAKDOWN_FIELDS,
+  SPAN_OP_RELATIVE_BREAKDOWN_FIELD,
+} from 'app/utils/discover/fields';
+import EventsTable from 'app/views/performance/transactionSummary/transactionEvents/eventsTable';
+
+type Data = {
+  features: string[];
+};
+
+function initializeData({features: additionalFeatures = []}: Data) {
+  const features = ['discover-basic', 'performance-view', ...additionalFeatures];
+  // @ts-expect-error
+  const organization = TestStubs.Organization({
+    features,
+    // @ts-expect-error
+    projects: [TestStubs.Project()],
+    apdexThreshold: 400,
+  });
+  const initialData = initializeOrg({
+    organization,
+    router: {
+      location: {
+        query: {
+          transaction: '/performance',
+          project: 1,
+          transactionCursor: '1:0:0',
+        },
+      },
+    },
+    project: 1,
+    projects: [],
+  });
+  ProjectsStore.loadInitialData(initialData.organization.projects);
+  return initialData;
+}
+
+describe('Performance GridEditable Table', function () {
+  let transactionsListTitles;
+  let fields;
+  let organization;
+  let data;
+  let transactionName;
+  const query =
+    'transaction.duration:<15m event.type:transaction transaction:/api/0/organizations/{organization_slug}/eventsv2/';
+  beforeEach(function () {
+    transactionName = 'transactionName';
+    transactionsListTitles = [
+      t('event id'),
+      t('user'),
+      t('operation duration'),
+      t('total duration'),
+      t('trace id'),
+      t('timestamp'),
+    ];
+    fields = [
+      'id',
+      'user.display',
+      SPAN_OP_RELATIVE_BREAKDOWN_FIELD,
+      'transaction.duration',
+      'trace',
+      'timestamp',
+      'spans.total.time',
+      ...SPAN_OP_BREAKDOWN_FIELDS,
+    ];
+    // @ts-expect-error
```

Comment:
@dashed Do we have an expected timeline of when we'll be able to remove these comments from tests?